src/scripts/main.js

```diff
@@ -1,3 +1,134 @@
 'use strict';
 
-// write code here
+function appendRow(button) {
+  const table = document.querySelector('.field > tbody');
+  const existRow = table.querySelector('tr');
+  const newRow = document.createElement('tr');
+
+  if (table.querySelectorAll('tr').length >= 10) {
+    button.disabled = true;
+
+    return 0;
+  }
+
+  newRow.innerHTML = existRow.innerHTML;
+
+  table.appendChild(newRow);
+
+  toggleButtons();
+}
+
+function removeRow(button) {
+  const table = document.querySelector('.field > tbody');
+  const rowToDel = table.querySelector('tr');
```

Comment:
The `removeRow` function always removes the first row. If the intention is to remove a specific row, you should modify the logic to target the correct row.